xbmc/ServiceManager.h

```diff
@@ -153,6 +157,7 @@ class CServiceManager
   std::unique_ptr<CPlatform> m_Platform;
   std::unique_ptr<PLAYLIST::CPlayListPlayer> m_playlistPlayer;
   std::unique_ptr<CSettings> m_settings;
+  std::unique_ptr<KODI::GAME::CControllerManager> m_controllerManager;
```

Comment:
Oh, I see what you're saying. Just rename it. I could do that, but it would be redundant - Anything outside of game should reference this as `GAME::CControllerManager`